dist/robots.cfg

```diff
@@ -58,6 +65,8 @@ RobotUA <<EOR
     gonzo,
     Google-Sitemaps,
     GoogleBot,
+    GPTBot
+    gptbot
```

Comment:
Match is case-insensitive, so delete one of these.

Reply:
done